alerts.md

```diff
@@ -2,42 +2,64 @@
 
 ### Summary
 
-Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via the Kubecost UI, and now via Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
+Kubecost alerts allow teams to receive updates on real-time Kubernetes spend. They are configurable via Helm values. This resource gives an overview of how to configure Kubecost email and Slack alerts using [Kubecost helm chart values](https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml).
```

Comment:
most are configurable in UI... why remove?

Reply:
These Go scheduled alerts are not configurable in the UI because the configmap overwrites their values. These recent changes have decoupled the new alerts from the cron scheduled ones. A new Notifications page would probably be necessary in the future.

I can put it back if we plan to keep the cron alerts active still.

Reply:
You're saying that users no longer have the ability to configure alerts in the UI??

Reply:
I apologize for the confusion, I just meant they won't configure be able to configure new alerts in the UI. The cron alerts are still there and configurable. I will add that back in and clarify in the doc.